packages/gnome_settings_daemon.rb

```diff
@@ -0,0 +1,74 @@
+# Deprecated: Newer versions are no longer compatible with Chromebrew due to dependence on systemd.
+
+require 'package'
+
+class Gnome_settings_daemon < Package
+  description 'GNOME Settings Daemon'
+  homepage 'https://gitlab.gnome.org/GNOME/gnome-settings-daemon'
+  version '43.0'
+  license 'GPL-2+ and LGPL-2+'
+  compatibility 'x86_64 aarch64 armv7l' # not compatible with i686 upstream
+  source_url 'https://gitlab.gnome.org/GNOME/gnome-settings-daemon.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
+
+  deprecated
```

Comment:
Just take this out.

Reply:
To explain this a bit better, we currently don't have a `depecated` property so not sure why this was added.

Reply:
It was part of the discussion.... But agree as per discussion it isn't needed at this time.

Reply:
Added to packages.rb